config/sync/paragraphs.paragraphs_type.recommendation.yml

```diff
@@ -0,0 +1,12 @@
o5XHpwzsDpcY1/xhhnOCpAiA1nqX2cc6Guk3FBvQLZvVIKQtyHLlmyRONkpR6oF9pk18ZNmwW4BwSyK005iPIsX/n548rhR8igxmCr3vjau7fNtap9XLP1NeWuUQCLermOOvTGOy4kuzQ2cAGZIztiYKAVNsUNpFP8RDqWa5ZEW5msBxpqRflRFkqJpvOI51GMO4nlDLDF+i4G4KhPWZUfEIdKUYBNf2OjB11Rbq8A2/IIsJmVY/8wnoOUDJCE+VU4T3idusLbwuvpjzfRniBfsjrj+8INjO+Y3A5vly0mP1JirlRtT8w19bHC+VnPtO5kZcPbZrKucN2Zrh+NNkSALdPeiPIdSQtMF7D5Lpoybd+/ErCh5nrpy17hbrjhr5Oa29LSbDGAiL26YcPG+H5/ws93lXjuuefILfiv4vnCCyWllfFN4cmm8KRiOjeFqVxZ+eANBRSTrKmS7yLt0ks/7N733ndbVr+Oj3/8o+6Sdyavgwn2xz/hTCuTDWALv9X+t7/+2T3ucRMQzW/v9u07FNMqpoxiO9CECT1S2F5+n1iGQ//4gO2zn/mUe8tb3uzvtmWULChagjRdwIYCKlW8QH5XMPEbpOuuu9ad9ZQn22p4MjU8NoIywedU5+5PcmcD2MJ9Exxvqs90gG064zidPcLxf4K4PcTvClSm73/fewwwE6t5xcpj482/4S/Xu8c//gxbn8qG8UKHFnybhhW3ft50gG0m7o0+ZCL20hvf+Hr/8PHlIwXYOAAw83vfu8qhZAncr8YPrIUwyM7mHvmWi9+muq+26sKxCjM59zBgS3VfmC5gm4lzyaaOdN8ZGInfxF/56tccyrpkKRyn7nC/75Idw9/mXyfTvR9OdW36xwjf32jLJ674VFxhu3TphCtRH3CHv6uPxL0v1djz7ZVqOXwPDOfDPSwvDPCZLH3841c43BKHE/kXLJhvc3GBgpM82QC2TMcv8xOlZRNKyvD3WgTYwr0TrT+WLBABtmn2Bhd+tinTMpnkm+roaetI0fZUZZJtT7YNe4S3h9czyZOseX49MdXEhPX9feH6w/vUwHjB8D5/PXE5XiTt+fntSiyfeMwjAdi27dpvCrbuvjG3YTMxugpdUQlQRfGqBMSKFYOqpKRc4EDKIAEHJvYBQIAUVDHEUMM0gAraTlB1VD9ANQAVwGxEChcUX0z6M2eLGgZYw4T+BGATHFAiH5OJqJnMLqoTmIGSKl/Hr62tcw3y31xb1yBIIthQUe2aBDEKBf52yp0e0AsodHB4yNoDYMS9IG3jD8gFBAQacWzagEINBRYwj3YB3IAcqLZoS1NTo+sWxEBt9uCDD6oNsy1fMcoouWVsa9srt5j9BhtwwVhX3yDQVWWKsvXrN1j7UBkR8wuYhRpqHOBYmOe2ynUj7iD5sUBbcFlJ++YLpLEtgGytcutYJED2uNNPV5mt5uYRJRbtbWlpcc3NzdY3PBzRR6jh6mSrTVs223lxHsQzGxAoLFE7gG4yiPLlCFpWWV9yzrjbbNvban3MOrZEATYEeFS/0j8qZv1u8FNAyewgF4W41URJRt8DZgBxQFVGMfuoG4WWqQX1GRsTMZeTBtg0NrA9sduIrVYul5DCOGo/KkkpuKRoKy8BAJdqnOYJpgnIjcmVpc4LuDQ8zJHypM7rE3TSeRbjYrJP/VFrSjNi4QE9GKP19fUGjTmXDinFKiorBCF7bKwcHJIasq1DNsL947i5IWrXOsCNMQ9U4/wgamqy2VsyLdWXJ3tprGps5asPeENtgfqxoWGO3IoukYvSRqkbu+Suc7v6P9dVzK6xsRcDayJXgtLYiWMQP7BVcG2fjtuhfh4bJ/5alZtTXWwx5riG9uqNTtyG9mm8sE57TJWofsIVJG1EzSavmja2GANYiE+UZSyjaqUs16wBOQwim2v4xACbypORawVQnAjYuKYgbnKzmdMTATbZ1U8RYPOtMXmZcZhJyjSfX9d0yvjl/eWZrMuvN5vlo9GGCLBl0yNR3iNlgekCNn+im5hXf0qh6PLbzYsmZXpxJ0gfkctF3GiR0k3SBflTfYYnzZLFg+EN7/e89/3xKnC9BGxChcEzHOnWW//hnvb0Z8Xz+BNn/mRfWLESL6AFnnln1zTEN4Unm8KTirt3bUvp5stXn+GCE1dxJB+8pXMzyDO4P/F11fe+684/P+ZWzZ/ETqd88dUA2QA2XKO3LFoWt8OvfvlzxZR7enzdX/Bd/rHdVxyE7eX3iV/H0QRsjJnb/nGzf/j48sMPP+xOPS0GB9l4773/NI8KLIcnU8Njgzx+murc/TEZnoBOdyxc4+NCLUjfv/p77pxzzgpWU37ilj+4VsJtm844TtfGsCs3FJD83sQ9KolrEHUZ6rWZGD/BSaezaZCHz3SAbSbujf71+a8CbMH58sLnL3/166TK3y998QuK03ehZX3Xu9/rvvGNb9lyumuEDL6b2wsueK376le+ZOX4J5NzP9KAbSbOxR9Lh/OdETeMFngx9UqBya9+9es2Z+Hvu/22W9zxx8eUqYf7fefXm2zZP7d0fZ3ufhi+h4Tv7f4xwvc32hSG8Jz/MccsS/j+9V0rU+ZI3PvC7eY4mabwPZBygP1Alco6kPDXv7omfu9lG8l/+YV1YuhefvmlupefbWCNbcxvVFTWsGgpG8Dmx5lMN37DbkzD32sRYAusH30+Fi0QAbZp9go/MLJNmZbJJN9UR09bR4q2pyqTbHuybdgjvD28nkmeZM3z6zFg4xnf3xeuP7zPqMKhsuF9/nri8sTB/O2TjmVT3bG8fr7w8pEBbJ1SsO1wvRJrbdqy2xRsKIPKyspdTU2dxbMalQpGXMWABA+2/GBHfYQSDbvIg5xNvqOCQUnDNlzz4QLQ4jfpEzd6xEdDBUQ/8CUbV81onhUYQH1AJiamzdWd8jKxCMQD1JRJ1VVWUiZXeMVSrM0T+JovNwTzpeiptB86gBZilQGGhgX1+OHVPzAkqFVhCiyDBKo/Xz+AgE
```

Comment:
Please make sure that this file has been compressed and made as small as is tolerable

On BUPL, we had a big issue that the whole site started crashing cause we had used too big images in the config files :)

I tend to take screenshots and turn them into JPGs and then compress them using

https://png2jpg.com/

Reply:
it's compressed now